### DISCUSSION TITLE
Can't resolve '.keystone/api'

### DISCUSSION THREAD

**Discussion by alvaaz** [keystonejs/keystone#6878] [Questions] [answered]
Hi!

I'm following [this](https://keystonejs.com/docs/walkthroughs/embedded-mode-with-sqlite-nextjs) example and Next can't resolve List. Can someone help me?

```
import { lists } from '.keystone/api';

export default function ShortIdPage() {
  return <div>ShortID Redirect</div>;
}

export async function getServerSideProps(data2) {
  console.log(data2, 'data');
  const data = await lists.Redirect.findUnique({
    query: `slug`,
  });

  if (!data) {
    return {
      redirect: { destination: '/' },
    };
  }

  return {
    redirect: {
      destination: data.url,
    },
  };
}
```
Structure folder
https://share.getcloudapp.com/YEuPXExX

Error
https://share.getcloudapp.com/jkuEekB8

**Comment by bladey** [comment, marked as answer]
Hey @alvaaz,

GitHub discussions is currently used for feature requests only, could you please ask your question in the Keystone Community Slack located here - https://community.keystonejs.com

Thank you!
